shift4/cards.py

```diff
@@ -2,14 +2,20 @@
 
 
 class Cards(Resource):
-    def create(self, customer_id, params):
-        return self._post("/customers/%s/cards" % customer_id, params)
+    def create(self, customer_id, params, request_options=None):
+        return self._post(
+            "/customers/%s/cards" % customer_id, params, request_options=request_options
+        )
 
     def get(self, customer_id, card_id):
         return self._get("/customers/%s/cards/%s" % (customer_id, card_id))
 
-    def update(self, customer_id, card_id, params):
-        return self._post("/customers/%s/cards/%s" % (customer_id, card_id), params)
+    def update(self, customer_id, card_id, params, request_options=None):
+        return self._post(
+            "/customers/%s/cards/%s" % (customer_id, card_id),
+            params,
+            request_options=request_options,
```

Comment:
Redundant comma at the end

Reply:
no such thing in python, but changed ;) 
https://peps.python.org/pep-0008/#when-to-use-trailing-commas

Reply:
I take that back, reverted as the Style Check requires it there.